core/libdeno/internal.h

```diff
@@ -25,6 +25,56 @@ struct ModuleInfo {
   }
 };
 
+class DenoArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
```

Comment:
Would be nice to put this in a separate file... everything is kinda slopped together into binding.cc and internal.h... I've been slowly trying to break it up as I add stuff

Reply:
s/DenoArrayBufferAllocator/ArrayBufferAllocator/g